roles/models.py

```diff
@@ -0,0 +1,45 @@
+from django.db import models
```

Comment:
Looking at the entirety of this file...why not use the models in `django.contrib.auth` [ https://docs.djangoproject.com/en/1.8/ref/contrib/auth/ ].

Even for the custom user model, the closer you stick to the abstract bases / mixins Django provides, the better.

In my view, this file consists of some entirely unnecessary re-invention - or, if necessary, not completely thought through.